libs/fetcher.client.js

```diff
@@ -290,7 +290,7 @@ Queue.prototype = {
          * @private
          */
         _constructGetUri: function (uri, resource, params, config) {
-            var query = [], matrix = [], id_param = config.id_param, id_val, final_uri = uri + '/resource/' + resource;
+            var query = [], matrix = [], id_param = config.id_param, id_val, final_uri = uri + '/' + resource;
```

Comment:
In `tests/unit/libs/fetcher.js` make sure to replace all instances of `/resource/` with `/`. That should solve our problems.